src/app/copmonents/header/header.component.scss

```diff
@@ -0,0 +1,33 @@
+.header {
+
+  height: 60px;
+  background-color: #E5EBF0;
```

Comment:
Good practice to use scss vars for colors. We understand what is this color used for.